plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -654,6 +658,54 @@ public String interpret(final BufferedReader reader) throws IOException {
         }
     }
 
+    /**
+     * setLiveMigrateFlags
+     *
+     * Sets the user configured live migration flags
+     *
+     * @param params
+     */
+    public void setLiveMigrateFlags(final Map<String, String> params) {
+        if (params.get(KVM_VM_MIGRATE_SPEED) != null) {
+            _migrateSpeed = NumbersUtil.parseInt(params.get(KVM_VM_MIGRATE_SPEED), -1);
```

Comment:
What about `NumbersUtil.parseInt(params.get(KVM_VM_MIGRATE_SPEED), getDefaultMigrateSpeed());` instead? The next `if` block won't be needed

Reply:
changed